### PR TITLE
 screencopy: record video of screenshots on failure

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

File: mir-ci/mir_ci/tests/robot/platforms/wayland/Screencopy.py [hunk 1]
```diff
@@ -137,6 +144,37 @@ def _log_failed_match(self, screenshot, template):
             html=True,
         )
 
+    def _end_suite(self, data, result) -> None:
+        if not result.passed:
+            video_path = f"{self._screenshots_dir}/video.webm"
+            try:
+                subprocess.run(
```
Comment (mattkae) [inline mir-ci/mir_ci/tests/robot/platforms/wayland/Screencopy.py:151]: Any reason to not just use `ffmpeg-python` to keep this in process? Although I am fine either way

**Review by mattkae — CHANGES_REQUESTED** [review]

File: mir-ci/mir_ci/tests/robot/platforms/wayland/Screencopy.py (hunk 1 above)
Comment (Saviq) [reply mir-ci/mir_ci/tests/robot/platforms/wayland/Screencopy.py:151]: Check what python-ffmpeg actually is ;)

In theory we could use `av`, but this is just simpler.

Comment (mattkae) [reply mir-ci/mir_ci/tests/robot/platforms/wayland/Screencopy.py:151]: Ah it just wraps a subprocess call, doesn't it... XD Okay you win. I think this is simple

**Review by mattkae — APPROVED** [review]
Easy enough :+1:

File: mir-ci/mir_ci/tests/robot/platforms/wayland/Screencopy.py (hunk 1 above)
Comment (Saviq) [reply mir-ci/mir_ci/tests/robot/platforms/wayland/Screencopy.py:151]: It's in the name, ffmpeg is the executable, libav is the library.

**Pull request merged**